matching-web/src/main/java/com/graphhopper/matching/cli/MatchCommand.java

```diff
@@ -59,23 +57,29 @@ public void configure(Subparser subparser) {
                 .type(Double.class)
                 .required(false)
                 .setDefault(2.0);
+        subparser.addArgument("--vehicle")
```

Comment:
This is a bit ugly: If we use vehicle=bike during import we have to use it also for the match command, even though in theory this information is already contained in the graph-cache folder. The problem is that we cannot simply use GraphHopper#load to obtain the encoding manager (and find out which vehicle was used), because to do this we need to set the profiles (because of `checkProfilesConsistency()`), but to set the profiles we need the vehicle.

Don't know the real ugliness is once again that `GraphHopper` does way too much. There certainly should be a way to e.g. load the encoding manager without specifying the profiles (which are only really needed for routing (or map matching here)), but so far I thought we can postpone this until https://github.com/graphhopper/graphhopper/issues/1895 and https://github.com/graphhopper/graphhopper/pull/1901. Not sure is it too ugly to add  the parameter here? If yes it should not be too hard to find somewhat better solution.

Reply:
This does not look to me too bad. If you do not like it we could dynamically pick the car via hopper.getEncodingManager().fetchEdgeEncoders().get(0) and create the profile?

Reply:
The loading is the problem: How can we load the encoding manager without setting the profiles? Right now we cannot do it via `GraphHopper#load` unless we use some workaround like loading with some dummy profiles and then loading again, but that all seems a bit weird. 

But we can just load the EncodingManager without GraphHopper? 
```java
EncodingManage.create(new DefaultEncodedValueFactory(), new DefaultFlagEncoderFactory(), ghLocation).fetchEdgeEncoders().get(0);
```

I did that in the last commit. I also fixed the case when using multiple vehicles during import (comma separated list) and its now possible to use `--vehicle` to select one of the (but no longer required since the first encoder is the default)

Reply:
> But we can just load the EncodingManager without GraphHopper?

Yes, that was what I meant. This is also used for Android

> but no longer required since the first encoder is the default

:+1: